src/views/SettingsSecurity.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+  <div class="wrapper">
+    <SettingsNav />
+    <SettingsHeader title="security" />
+    <button @click="showActivityFunc">login activity</button>
+
+    <section class="log" v-if="showActivity">
+      <h3>konza, kenya</h3>
+      <p class="status green">active now</p>
+      <p>Ulefone note 8p</p>
+    </section>
+
+    <section class="log" v-if="showActivity">
+      <h3>konza, kenya</h3>
+      <p class="status">22 july 2020</p>
+      <p>Ulefone note 8p</p>
+    </section>
+
+    <button @click="showColleaguesFunc">colleagues</button>
+
+    <section class="log" v-if="showColleagues">
```

Comment:
Lägg gärna in mer realistisk information. Dessa fälten ska ha namn och efternamn på kollegor inte bara förnamn och plats.

Reply:
när det gäller info så har jag lagt precis samma som ståd på mockupen, kanpparna kan jag fixa nu

Reply:
Det är striktare med informationen man använder i denna inlämningsuppgiften vs den vi skulle ha i mockupen. I denna uppgiften var David väldigt tydlig med att allt på sidan ska vara riktig information( eller så riktig den kan bli i alla fall) så vi måste göra lite annorlunda från hur det är i Figma-mockupen.